devbox.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packages": [
+    "yq-go@latest",
+    "kubebuilder@latest",
```

Comment:
I think we should pin the version of kubebuilder

Reply:
Agreed, seems lock is at `4.1.1`

on the other hand, anything we pin we need to revisit every now and then. It would be good to discuss how we make sure we do not forget to maintain the pinned versions on the longer run.